2020/13xxx/CVE-2020-13670.json

```diff
@@ -1,18 +1,74 @@
 {
```

Comment:
{} is not allowed for {'vendor': {'vendor_data': [{'product': {'product_data': [{'product_name': 'Core', 'version': {'version_data': [{'version_affected': '<', 'version_name': '8.8.x', 'version_value': '8.8.10'}, {'version_affected': '<', 'version_name': '8.9.x', 'version_value': '8.9.6'}, {'version_affected': '<', 'version_name': '9.0.x', 'version_value': '9.0.6'}]}}]}, 'vendor_name': 'Drupal'}]}}